docs/getting-started/built-ins.md

```diff
@@ -153,9 +153,75 @@ rshift(1024n, 11n) // 0n
 
 - `exit(status: boolean): void` Call this function will terminate contract execution. If `status` is `true` then the contract succeeds; otherwise, it fails.
 
+## `SmartContract` Functions
+
+The following functions come with the `SmartContract` base class.
+
+### `this.buildStateOutput(amount: bigint)`
+
+Function `this.buildStateOutput` creates an output containing the latest state. It takes an input: the number of satoshis in the output. With this function, you don't need to construct the new state output manually in the contract.
```

Comment:
Delete. Users cannot do it manually. So redundant.
`With this function, you don't need to construct the new state output manually in the contract.`